explain-flags.html

```diff
@@ -20,10 +20,15 @@
     function explainFlags() {
         var flagValue = parseInt(document.getElementById('tb').value); //returns 0 or NaN if can't parse
         var summary = "";
+        var pairedRead = lstFlags[0][1] & flagValue;
         for(var i = 0; i < lstFlags.length; i++) {
             var checkbox = document.getElementById('cb' + i)
             if(lstFlags[i][1] & flagValue) {
-                summary  += " &nbsp; &nbsp; " + lstFlags[i][0] + "<br>";
+                if (!pairedRead && i < 8){
+                    summary = " &nbsp; &nbsp; <b>Invalid flag: read not paired</b><br>";
```

Comment:
even though it isn't valid, I would still like to be able to see the flag...

Reply:
instead of hiding the summary in the event of an invalid flag, could you add a note to the end explaining ?
perhaps a summary of issue:
```
- Flag XXX cannot be set when read is not paired"
- ....
```

Reply:
I agree, this is a much better solution.